app/models/tutoring_session.rb

```diff
@@ -2,9 +2,11 @@ class TutoringSession < ApplicationRecord
   belongs_to :student
   belongs_to :tutor
 
-  validates :location,     presence: true
-  validates :start_time,   presence: true
-  validates :end_time,     presence: true
+  validates :location, presence: true
+  validates :hours, presence: true,
+                    length: { maximum: 4 }
```

Comment:
There are 744 hours in a 31 day month. I think at the very most this should validate as maximum of 744. More realistically we should ask Roberta for a better number... Just to prevent typos.